functions/vercel/api/query/[table].ts

```diff
@@ -0,0 +1,51 @@
+import type { VercelRequest, VercelResponse } from '@vercel/node'
+import type { TableNames } from '../../_lib/types'
+import 'isomorphic-fetch'
+import { getXataClient } from '../../_lib/xata.codegen'
+
+export default async function handler(
+  request: VercelRequest,
+  response: VercelResponse
+) {
+  const xata = getXataClient()
+  const table = request.query.table as TableNames
+
+  const { filter, sort } = request.body ?? {}
+
+  const hasFilter = !!filter
```

Comment:
```suggestion
  const hasFilter = Boolean(filter)
```

I usually prefer the `Boolean()` syntax, a bit more semantic